albo.py

```diff
@@ -14,11 +15,42 @@ def escape_char (text: str, char_to_escape: List[str] = ['_', '*', '[', '`']) ->
 def send_telegram_message(text: str) -> None:
     params = {
         'chat_id': CHATID,
-        'text': text
+        'text': text,
+        'parse_mode': 'markdown'
     }
-    URL = "https://api.telegram.org/bot" + TOKEN + "/sendMessage?&parse_mode=markdown&" + urlencode(params, quote_via=quote_plus)
+    URL = "https://api.telegram.org/bot" + TOKEN + "/sendMessage?&" + urlencode(params, quote_via=quote_plus)
     r = requests.get(url = URL)
 
+def send_single_telegram_attachment(pdf_link: str) -> None:
+    params = {
+        'chat_id': CHATID,
+        'document': pdf_link,
+        'disable_notification': True
+    }
+    URL = "https://api.telegram.org/bot" + TOKEN + "/sendDocument?&" + urlencode(params, quote_via=quote_plus)
+    r = requests.get(url = URL)
+
+def send_multiple_telegram_attachments(pdf_links: List[str]) -> None:
+    params = {
+        'chat_id': CHATID,
+        'disable_notification': True,
+        'media': json.dumps([
+          { 
+            "type": "document",
+            "media": pdf_link
+          }
+        for pdf_link in pdf_links]),
```

Comment:
this for seems a bit random here, move it outside the dictionary and save it in a separate variable please

(good job for the rest! 🚀 )